Packs/Core/Playbooks/playbook-Identity_Analytics_-_Alert_Handling_README.md

```diff
@@ -1,13 +1,13 @@
 The `Identity Analytics - Alert Handling` playbook is designed to handle Identity Analytics alerts and executes the following:
 
 Analysis:
-Enriches the IP and the account, providing additional context and information about these indicators.
+- Enriches the IP and the account, providing additional context and information about these indicators.
 
 Verdict:
-Determines the appropriate verdict based on the data collected from the enrichment phase.
+- Determines the appropriate verdict based on the data collected from the enrichment phase.
```

Comment:
When there is only one item in a bulleted list, no bullet is required.  In both these cases, the bullet should be removed as was in the original text.

Reply:
done